catalog/task/push-snapshot/0.4/README.md

```diff
@@ -0,0 +1,23 @@
+# push-snapshot
+
+Tekton task to push snapshot images to an image registry using `skopeo copy`.
+
+## Parameters
+
+| Name | Description | Optional | Default value |
+|------|-------------|----------|---------------|
+| mappedSnapshot | JSON string representing the Snapshot | No | - |
+| retries | Retry copy N times | Yes | 0 |
```

Comment:
Please add the tag parameter above this line here.

Also, please add a changelog entry below.

Reply:
fixed. thanks